tests/func.sh

```diff
@@ -1,45 +1,51 @@
 #!/bin/bash
 
-
 check_outputs () {
-    local readonly command="$@"
-    local readonly hashed=$(echo -n "${command}" | md5sum | cut -d" " -f1)
+    command=( "$@" )
+    hashed=$(echo -n "${command[@]}" | md5sum | cut -d" " -f1)
 
-    ${command} > orig_${hashed}_stdout.txt 2> orig_${hashed}_stderr.txt
-    echo "$?" > orig_${hashed}_exitcode.txt
+    # shellcheck disable=SC2068
```

Comment:
Disabled check is for ```Double quote array expansions to avoid re-splitting elements.```
https://github.com/koalaman/shellcheck/wiki/SC2068

When i apply change proposed on page, script stop working.